schema/release-schema.json

```diff
@@ -2135,6 +2155,41 @@
         }
       }
     },
+    "SimpleUnit": {
+      "title": "Simple Unit",
+      "description": "A simple unit consists of: an identifier for the list (scheme) from which the unit is taken, and an identifier for the category from that list being applied.",
+      "type": "object",
+      "properties": {
+        "scheme": {
+          "title": "Scheme",
+          "description": "The list from which identifiers for units of measure are taken, using the open [unitClassificationScheme](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#unit-classification-scheme) codelist. 'UNCEFACT' is recommended.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "codelist": "unitClassificationScheme.csv",
+          "openCodelist": true
+        },
+        "id": {
+          "title": "ID",
+          "description": "The unit code taken from the scheme. Although an integer is allowed, it is recommended to use a string.",
+          "type": [
+            "string",
+            "integer",
+            "null"
+          ],
+          "versionId": true
+        }
+      },
+      "patternProperties": {
+        "^(description_(((([A-Za-z]{2,3}(-([A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-([A-Za-z]{4}))?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-([0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(x(-[A-Za-z0-9]{1,8})+))?)|(x(-[A-Za-z0-9]{1,8})+)))$": {
```

Comment:
Are `patternProperties` necessary? I don't see a `description` field which can be translated.

Reply:
You are right!